test/unit/Application.test.js

```diff
@@ -20,16 +20,19 @@ describe('Application', () => {
   const context = mockRouterContext();
 
   before(() => {
-    window.matchMedia = () => ({ addListener: () => {} });
+    window.matchMedia = () => ({ addListener: () => undefined });
     window.matchMedia().addListener = stub();
     component = shallow(
       <Application
-        children={{}}
         route={{
           history: { listen: stub() },
         }}
-        addFeatures={() => {}}
-      />, { context });
+        addFeatures={() => undefined}
+      >
+        {}
```

Comment:
Ah sounds good. 

My advice would be to use this PR: https://github.com/NYPL/discovery-front-end/pull/1846 to just run prettier on everything.  It only makes PRs harder to review when you can't tell what's a functional change and what's because of the linter.

Reply:
I agree who heartedly, though sometimes circumstances prevent having a clean PR. This file, however, was an instance of oversight and was bundled up with into this PR. Not too hairy though.